modules/cnf-performing-end-to-end-tests-running-cyclictest.adoc

```diff
@@ -33,8 +28,7 @@ When executing `podman` commands as a non-root or non-privileged user, mounting
 [source,terminal,subs="attributes+"]
 ----
 $ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
--e LATENCY_TEST_RUN=true -e DISCOVERY_MODE=true -e FEATURES=performance -e ROLE_WORKER_CNF=worker-cnf \
--e LATENCY_TEST_CPUS=10 -e LATENCY_TEST_RUNTIME=600 -e MAXIMUM_LATENCY=20 \
+-e ROLE_WORKER_CNF=worker-cnf -e LATENCY_TEST_CPUS=10 -e LATENCY_TEST_RUNTIME=600 -e MAXIMUM_LATENCY=20 \
```

Comment:
ROLE_WORKER_CNF is not making any difference, I think this better be fixed in order to allow users to test on specific labeled nodes. I'll take this with @liornoy offline.

Reply:
discussed offline and aggreed to remove env vars that no longer add value to the testing. we agreed on adding them in the release notes.